vllm/worker/xpu_model_runner.py

```diff
@@ -109,9 +109,8 @@ def __init__(
 
         self.kv_cache_dtype = kv_cache_dtype
         self.block_size = cache_config.block_size
-        self.max_context_len_to_capture = (
-            self.model_config.max_context_len_to_capture
-            if self.model_config is not None else 0)
+        self.max_context_len_to_capture = (None if self.model_config
```

Comment:
It seems that this property is not being used at all, so I guess we can remove it entirely from `XPUModelRunner`.